cross-chain/arbitrum/deploy_l2/03_upgrade_arbitrum_wormhole_gateway.ts

```diff
@@ -0,0 +1,50 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const ArbitrumTokenBridge = await deployments.get("ArbitrumTokenBridge")
+  const ArbitrumWormholeTBTC = await deployments.get("ArbitrumWormholeTBTC")
+  const ArbitrumTBTC = await deployments.get("ArbitrumTBTC")
+
+  const [, proxyDeployment] = await helpers.upgrades.upgradeProxy(
+    "ArbitrumWormholeGateway",
+    "ArbitrumWormholeGateway",
+    {
+      contractName:
+        "@keep-network/tbtc-v2/contracts/l2/L2WormholeGateway.sol:L2WormholeGateway",
```

Comment:
We need to upgrade the `@keep-network/tbtc-v2` in the `package.json`/`yarn.lock` to fetch the latest version of the `L2WormholeGateway` contract.

Reply:
Yes, waiting for this guy to be merged https://github.com/keep-network/tbtc-v2/pull/616

Reply:
It's merged. Can we update the `yarn.lock` to point to the latest dev package?

Reply:
[69a6f40](https://github.com/keep-network/tbtc-v2/pull/617/commits/69a6f4076cc2b39e346dab0952e2f700427cb177)